pixi.toml

```diff
@@ -86,6 +81,9 @@ test-unit = 'pytest spatialpandas/tests -n logical --dist loadgroup'
 [feature.test-example.dependencies]
 nbval = "*"
 
+[feature.test-example.activation.env]
+DASK_SCHEDULER = "single-threaded"
```

Comment:
This is to try to avoid this kind of error (which also happens on main)

![image](https://github.com/user-attachments/assets/0ab508c0-a841-45a6-9605-3358e0f64e64)